unit/data/csv/row_example_comment.csv

```diff
@@ -0,0 +1,2 @@
+0,1,2,3#time
+0,4,5,6#amplitude
```

Comment:
And an end of line here. Also, in this file can we add some whitespace? 

I'm sure your method will handle it, but you can have comments on some lines and not others right? Just want to make sure. My example had a comment on just one line (like you might put in a script).

Reply:
Done.